src/overlays/actors/ovl_En_Bb/z_en_bb.c

```diff
@@ -631,38 +615,69 @@ s32 EnBb_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* po
     return false;
 }
 
+/**
+ * This maps a given limb based on its limbIndex to its appropriate index
+ * in the bodyPartsPos/Velocity arrays. An index of -1 indicates that the
+ * limb is not part of the bodyParts arrays.
+ */
+static s8 sLimbToBodyParts[BUBBLE_LIMB_MAX] = {
+    -1,                // BUBBLE_LIMB_NONE
+    -1,                // BUBBLE_LIMB_ROOT
+    -1,                // BUBBLE_LIMB_CRANIUM_ROOT
+    -1,                // BUBBLE_LIMB_JAW_ROOT
+    BUBBLE_BODYPART_0, // BUBBLE_LIMB_JAW
```

Comment:
It would make sense to have `BUBBLE_BODYPART_NONE = -1` for all these body parts, but I left it out for now just to see what it would look like since it does make the other body parts more readable in these `sLimbToBodyParts` . But I can add in the `-1` enums since that would be more standard